### PR TITLE
How to write different logs to different files and caching recipe

### PR DESCRIPTION
**Pull request by summerknight**

### CODE REVIEW

File: book/caching.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Caching
```
Comment (samdark) [inline book/caching.md]: Isn't it already covered well by official guide? https://github.com/yiisoft/yii2/blob/master/docs/guide/caching-data.md

File: book/logging-problems-and-solutions.md [hunk 2]
```diff
@@ -63,3 +63,38 @@ the progress. In this case you need to change settings via application config fi
     ]
 ]
 ```
+
+Write different logs to different files
```
Comment (samdark) [inline book/logging-problems-and-solutions.md]: Makes sense to have it. Probably `logging-problems-and-solutions.md` should be separated into individual small recipes.

File: book/caching.md (hunk 1 above)
Comment (summerknight) [inline book/caching.md]: In the guide there is a relatively big article, covers a lot of topics at once. This recipe is a concrete solution for concrete task. Small and concise.
So include or not - depends on cookbook concept.

Comment (samdark) [inline book/caching.md]: That's probably means guide should be divided into more smaller topics. But yeah, it makes sense to include it into cookbook as well.

**Comment by summerknight** [comment]
How to merge this pull request?

**Comment by samdark** [comment]
Done. Thank you!

**Pull request merged**